cipd_packages/README.md

```diff
@@ -0,0 +1,77 @@
+# Flutter CIPD Packages
+
+This folder contains auto built/deployed public [CIPD packages](https://chrome-infra-packages.appspot.com/p/flutter)
+used by Flutter CI. These packages will be consumed from tests and auto cached to boost future runs.
+
+Pleaes follow these steps to add a new package.
+
+## Creating the package build file
+
+Create a new sub directory under this folder named with the new package. Add a `build.sh` script for Linux/Mac or
+`build.bat` for Windows, and put it under a child dir `tool`. This is the main script to build this package. Add
+necessary lib files if needed.
+
+Note:
+
+1) we do not achive the binaries in the repo, and will auto build package artifacts and upload to CIPD via the bot.
+2) please add a code owner entry under section `cipd packages` in [CODEOWNERS](https://github.com/flutter/cocoon/blob/main/CODEOWNERS) file.
+
+## Add an entry to cocoon .ci.yaml
+
+This is to enable the auto build and upload. Please follow the following example format:
+```yaml
+  - name: Mac <package_name>
+    recipe: cocoon/cipd
+    bringup: true
+    properties:
+      script: cipd_packages/<package_name>/tool/build.sh
+      cipd_name: flutter/<package_name>/mac-amd64 # Use mac-arm64 for arm64 version.
+      device_type: none
+    runIf:
+      - cipd_packages/<package_name>/**
+      - .ci.yaml
+```
+
+Start with `bringup: true`, same as any other regular new target. Once validated in CI, remove it to enable running
+in the `prod` environment so that artifacts are to be uploaded to CIPD.
+
+Note: with `bringup: true`, the target will be executed in a staging environment and it validates only the logic
+and will not upload to CIPD.
+
+## Adding reference to the CIPD package
```

Comment:
nit: Adding a reference to the CIPD package

Reply:
Updated.